src/Parsers.jl

```diff
@@ -96,15 +96,20 @@ function Options(
             dateformat::Union{Nothing, String, Dates.DateFormat, Format},
             ignorerepeated, ignoreemptylines, comment, quoted, debug, stripwhitespace=false, stripquoted=false)
     asciival(wh1) && asciival(wh2) || throw(ArgumentError("whitespace characters must be ASCII"))
-    asciival(oq) && asciival(cq) && asciival(e) || throw(ArgumentError("openquotechar, closequotechar, and escapechar must be ASCII characters"))
+    (oq isa String || asciival(oq)) && (cq isa String || asciival(cq)) && asciival(e) || throw(ArgumentError("openquotechar, closequotechar, and escapechar must be ASCII characters"))
     (oq == delim) || (cq == delim) || (e == delim) && throw(ArgumentError("delim argument must be different than openquotechar, closequotechar, and escapechar arguments"))
```

Comment:
Need to think about how `delim` and `oq`/`cq` interact e.g. when `delim::Char` in `oq::String`

Reply:
we should probably just check the `delim` and `oq`/`cq` (i) are not equal, (ii) none of the chars in one are in the other